### ISSUE TITLE
Jump are treated as cuts

### ISSUE THREAD

**Issue by drepetto** [inkstitch/inkstitch#1879]
Hello, thanks for inkstitch, it's a great tool.

I just upgraded to 2.2.0 on OSX Monterey 12.6. Suddenly all jumps are being treated as cuts by the Brother PR1050x machine. When I use the visualizer they are jumps. But when I stitch it out on the machine they are treated as cuts. This is an edit of a file that I have stitched out before with no problems. Has something changed in the way jumps/cuts are saved? 

Thanks, Douglas

**Comment by tyrosinase** [comment]
I also feel like something has changed, but also my machine regularly resets itself to auto-trim so I've never been quite sure.

**Comment by drepetto** [comment]
Hmm, good idea, I will look at the auto-trim setting on the machine this evening. I'm pretty sure that's not it since the previous version of the file (saved with an older version of inkstitch) worked fine an hour before and the machine hasn't been restarted or anything...

**Comment by tyrosinase** [comment]
Mine resets whenever it's unplugged which has happened quite a bit lately, between moving it to a new stand and moving it to the table to oil/clean it, and the menu option is buried so deep that I have been leaving it on auto-trim more often than not... so while I thought I noticed a change, I wasn't sure if it was Ink/Stitch or me. Odds usually favor it being me, so I chalked it up to my imagination but now I think it may not be.

I also feel like there's been some chatter about a trim change in commits, but now I can't find any so I'm going to chalk that up to my imagination as well, until an actual dev chimes in.
